processor/batchprocessor/README.md

```diff
@@ -19,6 +19,9 @@ The following configuration options can be modified:
 batch will be sent.
 - `timeout` (default = 200ms): Time duration after which a batch will be sent
 regardless of size.
+- `enfoce_batch_size` (default = false): Ensures that batch size does not overflow `send_batch_size`.
```

Comment:
What do you think about https://github.com/open-telemetry/opentelemetry-collector/issues/1020, which proposes to have a hard limit instead of the boolean.

Reply:
I'd suggest this approach as well

Reply:
Having the hard upper limit (from #1020) is more flexible. Batches bigger just by a small number will pass whereas the very large ones will be split. 

I will make the change.

Reply:
Done, the last commit introduces `send_batch_max_size`